.circleci/config.yml

```diff
@@ -29,6 +29,27 @@ jobs:
                 circleci-agent step halt
             fi
 
+  check_main:
+    machine:
+      image: ubuntu-2004:202107-02 # Ubuntu 20.04, Docker v20.10.7, Docker Compose v1.29.2
```

Comment:
I see the blame for the other images is from a few years ago, would it make sense to open a ticket to look at moving to 22.04 at some point? 20.04 is LTS so it's definitely not pressing, just something I thought about.

Reply:
Oh yeah good idea 👍 https://github.com/mozilla/experimenter/issues/8839